pkg/cli/create_helm.go

```diff
@@ -365,6 +373,24 @@ func (cmd *createHelm) addVCluster(ctx context.Context, vClusterConfig *config.C
 	return nil
 }
 
+func (cmd *createHelm) isLoftAgentDeployed(ctx context.Context) (bool, error) {
+	podList, err := cmd.kubeClient.CoreV1().Pods("").List(ctx, metav1.ListOptions{
+		LabelSelector: "app=loft",
+	})
+	if err != nil && !kerrors.IsNotFound(err) {
+		return false, err
+	}
+
+	return len(podList.Items) > 0, nil
```

Comment:
Can we narrow this down by namespace?

Reply:
Since people can deploy loft in any namespace, I'm not sure if this would be a good idea